apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -258,21 +257,40 @@ public Bundle searchByCoverageContract(
     CcwCodebookVariable partDContractMonth = partDCwVariableFor(contractMonth);
     SingularAttribute<Beneficiary, String> contractMonthField = partDFieldFor(partDContractMonth);
 
+    PageQueryBuilder<Beneficiary> queryBuilder =
+        new PageQueryBuilder<Beneficiary>(Beneficiary.class, entityManager);
+
+    queryBuilder
```

Comment:
This type of statement where `queryBuilder` is using itself as input is hard for me to reason about.